GNUmakefile

```diff
@@ -372,7 +373,21 @@ protoc-install:
 		chmod +x $(PROTOC_ROOT)/bin/protoc ; \
 	fi
 
-proto: protoc-install $(PROTOGOFILES) $(PROTOGOBINFILES)
+.PHONY: proto
+proto: -protoc-files mog
+
+.PHONY: mog
+mog:
```

Comment:
The `proto` target operates on files, but the `mog` target operates on packages which meant that the `pbservice` build was running mog extra times.

Reply:
can we make mog dependent on proto here, so when we run mog target it proto first or does that mean that proto will regenerate the files every time?